src/js/containers/covid19/recipient/RecipientContainer.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * RecipientContainer.jsx
+ * Created by Jonathan Hill 06/08/20
+ */
+
+import React, { useState } from 'react';
+import { uniqueId } from 'lodash';
+import tabs from 'dataMapping/covid19/recipient/recipient';
+import RecipientTab from 'components/covid19/recipient/RecipientTab';
+import DateNote from 'components/covid19/DateNote';
+
+const RecipientContainer = () => {
+    const [activeTab, setActiveTab] = useState('recipient_locations');
+    return (
+        <div className="recipient__container">
+            <DateNote />
+            <div className="recipient__title">
```

Comment:
Could use the shared `body__narrative` class here

Reply:
?